datasets/blbooksgenre/README.md

```diff
@@ -0,0 +1,389 @@
+- "From Tonkin to India : by the sources of the Irawadi, January '95-January '96",
+- 'Case of the Ameers of Sinde : speeches of Mr. John Sullivan, and Captain William Eastwick, at a special court held at the India House, ... 26th January, 1844',
+- 'The Andaman Islands; their colonization, etc. A correspondence addressed to the India Office',
+- 'Ancient India as described by Ptolemy; being a translation of the chapters which describe India and Eastern Asia in the treatise on Geography written by Klaudios Ptolemaios ... with introduction, commentary, map of India according to Ptolemy, and ... index, by J. W. McCrindle']
+
+Searching form the string `Africa` in the titles and randomly sampling 10 titles returns:
+
+- ['De Benguella ás Terras de Iácca. Descripção de uma viagem na Africa Central e Occidental ... Expedição organisada nos annos de 1877-1880. Edição illustrada',
+- 'To the New Geographical Society of Edinburgh [An address on Africa by H. M. Stanley.]',
+- 'Diamonds and Gold in South Africa ... With maps, etc',
+- 'Missionary Travels and Researches in South Africa ... With notes by F. S. Arnot. With map and illustrations. New edition',
+- 'A Narrative of a Visit to the Mauritius and South Africa ... Illustrated by two maps, sixteen etchings and twenty-eight wood-cuts',
+- 'Side Lights on South Africa ... With a map, etc',
+- 'My Second Journey through Equatorial Africa ... in ... 1886 and 1887 ... Translated ... by M. J. A. Bergmann. With a map ... and ... illustrations, etc',
+- 'Missionary Travels and Researches in South Africa ... With portrait and fullpage illustrations',
+- '[African sketches.] Narrative of a residence in South Africa ... A new edition. To which is prefixed a biographical sketch of the author by J. Conder',
+- 'Lake Ngami; or, Explorations and discoveries during four years wandering in the wilds of South Western Africa ... With a map, and numerous illustrations, etc']
+
+Whilst this dataset doesn't include the underlying text it is important to consider the potential attitudes represented in the title of the books, or the full text if you are using this dataset in conjunction with the full text.
+
+[More Information Needed]
+
+### Other Known Limitations
+
+[More Information Needed]
+
+## Additional Information
+
+### Dataset Curators
+
+[More Information Needed]
+
+### Licensing Information
+
+The books are licensed under the [CC Public Domain Mark 1.0](https://creativecommons.org/publicdomain/mark/1.0/) license.
+
+### Citation Information
+
+@misc{british library_genre, 
+title={ 19th Century Books - metadata with additional crowdsourced annotations}, 
+url={https://doi.org/10.23636/BKHQ-0312},
+author={{British Library} and  Morris, Victoria and van Strien, Daniel and Tolfo, Giorgia and Afric, Lora and Robertson, Stewart and Tiney, Patricia and Dogterom, Annelies and Wollner, Ildi},
+year={2021}} 
```

Comment:
You can use a bibtex block
````suggestion
```bibtex
@misc{british library_genre, 
title={ 19th Century Books - metadata with additional crowdsourced annotations}, 
url={https://doi.org/10.23636/BKHQ-0312},
author={{British Library} and  Morris, Victoria and van Strien, Daniel and Tolfo, Giorgia and Afric, Lora and Robertson, Stewart and Tiney, Patricia and Dogterom, Annelies and Wollner, Ildi},
year={2021}}
```
````